src/utilities/_hypre_utilities.hpp

```diff
@@ -991,34 +988,27 @@ struct oop_axpy
  * x += y
  * for vectors x,y
  */
-template<typename T>
+template <typename T>
 struct xpy
 {
    typedef thrust::tuple<T, T, T> Tuple;
 
-   __host__ __device__ T operator()(const T& x, const T& y) const
-   {
-	    return x + y;
-   }
+   __host__ __device__ T operator()(const T &x, const T &y) const { return x + y; }
 };
 
-
 /**
  * @brief oop_xy
  *
  * Performs
  * o = x .* y
  * For vectors o,x,y,z
  */
-template<typename T>
+template <typename T>
 struct oop_xy
 {
-   typedef thrust::tuple<T&, T, T> Tuple;
+   typedef thrust::tuple<T &, T, T> Tuple;
```

Comment:
Possible to have right justified pointers, but left justified references when standalone?